src/viam/robot/client.py

```diff
@@ -738,25 +757,29 @@ async def discover_components(
         queries: List[DiscoveryQuery],
     ) -> List[Discovery]:
         """
-        Get the list of discovered component configurations.
+        Get a list of discovered potential component configurations.
```

Comment:
I can't say I understand what potential means in this context. Do you know? Can you explain?

Reply:
rand explained it to me, its basically a long list of all the discovered possible configurations for the camera. like height this many pixels width this many pixels this webcam path or this webcam path etc. can confirm thats what the printout looks like and she approved of this terminology